Rakefile

```diff
@@ -208,12 +189,13 @@ if File.exist?("tool/automatiek.rake")
 
   # We currently include the following changes over the official version:
   # * Avoid requiring the optional `net-http-pipeline` dependency, so that its version can be selected by end users.
+  # * Require vendored net/http version RubyGems if available, otherwise the stdlib version.
   desc "Vendor a specific version of net-http-persistent to bundler"
   Automatiek::RakeTask.new("net-http-persistent") do |lib|
     lib.version = "v4.0.2"
     lib.download = { github: "https://github.com/drbrain/net-http-persistent" }
     lib.namespace = "Net::HTTP::Persistent"
-    lib.prefix = "Bundler::Persistent"
+    lib.prefix = "Gem"
```

Comment:
Should it really be `Gem` prefix instead of `Bundler`?

Reply:
It was not only Bundler loading `net/http`, but also rubygems. The only way I managed to make net-http-persistent use the vendored copy in RubyGems when available was to put it under the `Gem` namespace.

It feels cleaner to me anyways, since it's a `net/http` extension after all and I believe this code organization makes that more clear.

Reply:
It got me by surprise, because it is located under `bundler/lib/bundler/vendor/` and therefore it should logically have the `Bundler` prefix. So I wanted to double check.

But since I struggled with the namespaces a bit to fix our [test case in Fedora](https://src.fedoraproject.org/rpms/ruby/blob/rawhide/f/ruby.spec#_949-956), I understand that this was compromise.

Thx for elaborating.